ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/431__data-science-pipelines-api.robot

```diff
@@ -40,6 +40,21 @@ Verify Ods Users Can Do Http Request That Must Be Redirected to Https
     Should Start With    ${url}    https
     Remove Pipeline Project    project-redirect-http
 
+Verify DSPO Operator Reconciliation Retry
+    [Documentation]    Verify DSPO Operator is able to recover from missing components during the initialization
```

Comment:
Can you explain better what this test is doing?

I'm guessing it's doing this, but not sure
`Verifies that, after creating a DataSciencePipelinesApplication with missing information, once the information is provided DSPO reconciles it automatically and performs needed actions`

Reply:
> once the information is provided DSPO reconciles it automatically

in practice, every two it will reconcile

Reply:
It was a proposal. Can you update the Documentation with the correct info?

Reply:
It is not accurate. The code that we are running in this Keyword is `Verify DSPO Operator is able to recover from missing components during the initialization`.